verticapy/vdataframe.py

```diff
@@ -1590,13 +1582,6 @@ def __vDataFrameSQL__(self, table: str, func: str, history: str):
     ---------------------------------------------------------------------------
     This method is to use to build a vDataFrame based on a relation
         """
-        check_types(
```

Comment:
decorator needed on top of fucntion line 1580

Reply:
not necessary i guess because this is an internal function

Reply:
yes